heidrun/TN_mods.rb

```diff
@@ -0,0 +1,144 @@
+Krikri::Mapper.define(:tn_mods, :parser => Krikri::ModsParser) do
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/tn'
+    label 'Tennessee Digital Library'
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('mods:recordInfo', 'mods:recordContentSource')
+  end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('mods:location', 'mods:url')
+         .match_attribute(:usage, 'primary display')
```

Comment:
Yes, this appears the case, per the [crosswalk document](https://docs.google.com/spreadsheets/d/1BzZvDOf4fgas3TD21xF40lu2pk2XW0k2pTGJKIt6438/edit#gid=1453046017).